superset/app.py

```diff
@@ -324,23 +323,11 @@ def init_views(self) -> None:
             category_label=__("Sources"),
             category_icon="fa-wrench",
         )
-        appbuilder.add_link(
-            "Tables",
-            label=__("Tables"),
-            href="/tablemodelview/list/?_flt_1_is_sqllab_view=y",
-            icon="fa-table",
-            category="Sources",
-            category_label=__("Sources"),
-            category_icon="fa-table",
-        )
 
         #
         # Conditionally setup log views
         #
-        if (
-            not self.config["FAB_ADD_SECURITY_VIEWS"] is False
-            or self.config["SUPERSET_LOG_VIEW"] is False
-        ):
+        if self.config["FAB_ADD_SECURITY_VIEWS"] and self.config["SUPERSET_LOG_VIEW"]:
```

Comment:
Logic is not the same as it was. If this was the previous logic, then the contrapositive would be correct: 
```
not (a is False or b is False) == a and b
```
but the above statement lacks the inner parens:
```
# The not negates the result of a is False
not a is False or b is False
```

Reply:
Oh right. Well then I'm fixing the logic. It's not super clear reading the code that the `or` operator precedes `not`.

But I'm thinking that if either of these settings are moved away from their default (of `True`), we should not show the log view.

Reply:
I think the original slightly weird double negative notation is a consequence of originally having assumed that config parameters weren't always set. This has changed with the adoption of square brackets when retrieving config params over `get()`. I think it makes sense to think critically of what the behavour should be instead of persisting the existing logic as-is.

Reply:
I totally agree that this logical statement was funky. Took me a while to unravel it myself. If you're good with just simplifying it and taking whatever behavior changes that come out of it, then I'm good with that too.